src/ansys/fluent/core/fluent_connection.py

```diff
@@ -404,7 +404,8 @@ def _close_slurm(self):
         subprocess.run(["scancel", f"{self._slurm_job_id}"])
 
     def force_exit(self):
-        """Immediately terminates the Fluent client, losing unsaved progress and data.
+        """Immediately terminates the Fluent client (stand-alone or running inside a
+        container), losing unsaved progress and data.
```

Comment:
`force_exit()` also works for sessions running in Slurm environments

Reply:
Will update the docstring. Thank you @raph-luc